build/Helix/PrepareHelixPayload.ps1

```diff
@@ -0,0 +1,35 @@
+$flavor = "Debug"
+$platform = "x86"
+$payloadDir = "HelixPayload"
+
+$repoDirectory = Join-Path (Split-Path -Parent $script:MyInvocation.MyCommand.Path) "..\..\"
+$nugetPackagesDir = Join-Path (Split-Path -Parent $script:MyInvocation.MyCommand.Path) "packages"
+ 
+# Create the payload directory. Remove it if it already exists.
+If(test-path $payloadDir)
+{
+    Remove-Item $payloadDir -Recurse
+}
+New-Item -ItemType Directory -Force -Path $payloadDir
+
+# Copy files from nuget packages
+copy "$nugetPackagesDir\microsoft.windows.apps.test.1.0.181203002\lib\netcoreapp2.1\*.dll" $payloadDir
```

Comment:
nit: Any powershell linter is probably going to complain that you're not using the "Copy-Item" task

Reply:
Thanks. I'll replace.